park-dashboard/src/Components/Searchbar/Searchbar.jsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import Select from "react-select";
+import { Link } from "react-router-dom";
+import "./searchbar.css";
+
+const Searchbar = ({ parks }) => {
+  const options = parks
+    .filter((park) => park.designation === "National Park")
+    .map((park) => ({
+      value: park.fullName,
+      label: (
+        <Link className="links" to={`/parkInfoPage/${park.parkCode}`}>
```

Comment:
Can we give this link to the dropdown field instead of a label to redirect to parkInfoPage?

Reply:
Sure, I can try this.

Reply:
Links are now added to the full dropdown field.